Dockerfile

```diff
@@ -28,28 +28,26 @@
 # version becomes the new default, if this ncov-ingest image itself is still
 # relevant at that time.
 #   -trs, 19 Jan 2020
-FROM nextstrain/base:branch-python-base
 
-# Install Python package for which Python 3.7 wheels do not yet exist on PyPI.
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        python3-netifaces \
-        time\
-        xz-utils
+# Let's drop dependency on an image of unknown origin and start over.
+#   -ivan.aksamentov 2021-12-13
+FROM python:3.7-buster
```

Comment:
I'm a little confused on historical context here, but maybe @tsibley's TODO comment can be removed.

Reply:
The [python-base has been merged](https://github.com/nextstrain/docker-base/pull/21), so I think we can just use `nextstrain/base:latest`.

Reply:
A-ha! The mystery of a missing container image solved! 
We don't need anything from the base image though, and pretty much anything other than python and pipenv.